vic/drivers/classic/src/set_output_defaults.c

```diff
@@ -52,9 +52,7 @@ set_output_defaults(out_data_struct *out_data)
     if (options.LAKES) {
         options.Noutfiles++;
     }
-    out_data_files = (out_data_file_struct *) calloc(options.Noutfiles,
-                                                     sizeof(
-                                                         out_data_file_struct));
+    out_data_files = calloc(options.Noutfiles, sizeof(out_data_file_struct));
```

Comment:
right hand side should read `sizeof(*out_data_files)`. We are trying to get rid of the type info in the `calloc()` statement.